common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1679,6 +1687,16 @@ public enum API {
          */
         public static final String GET_SSO_TOKEN_PATH = "/ssoToken";
 
+        /**
+         * Broker api path to check if QR + PIN should de available.
```

Comment:
Content Provider path?

Perhaps that should be reflected in the javadoc

Reply:
updated